SPECS/opam.spec

```
@@ -1,15 +1,15 @@
 %global package_speccommit f40573a915953b3b1a24591e90b15a1bb9a8ff80
-%global usver 2.1.4
-%global xsver 4
+%global usver 2.4.1
+%global xsver 1
```

We may end up using an unmodified `2.4.1-1` from XS, so maybe:

```suggestion
%global usver 2.4.1
%global xsver 0
```

... plus:
```suggestion
Version:        2.4.1
Release:        %{?xsrel}.1%{?dist}
```

---

I don't disagree, but I think even if we don't anticipate that, when/if we rebase on a XenServer 2.4.1-1 package, unless we want to throw away our own changelog history (and that's something we usually don't want for maintenance updates. We can restart fresh for new XCP-ng release instead), we'd number it 2.4.1-1.1 anyway.

So both Pau's version and your proposal look fine to me.

---

Seeing that the release can still work even if XS releases their own version, I've added the .1 to the release

---

Not sure how much nitpicking it is, but isn't `-1.1` here would imply this is a modification of an upstream `-1`, which is why I suggested `-0.1` in my original comment ?